src/streams/EditStreamCard.js

```diff
@@ -50,10 +46,18 @@ export default function EditStreamCard(props: Props): Node {
     setDescription(description);
   }, []);
 
-  const handlePrivacyChange = useCallback(isPrivate => {
-    setPrivacy(isPrivate ? 'private' : 'public');
+  const handlePrivacyChange = useCallback((privacy: Privacy) => {
+    setPrivacy(privacy);
   }, []);
 
+  const privacyOptions = useMemo<$ReadOnlyArray<{| key: Privacy, title: LocalizableText |}>>(
+    () => [
+      { key: 'public', title: 'Public' },
+      { key: 'private', title: 'Private' },
```

Comment:
I was wrong in the office today saying 'Public' and 'Private' aren't in messages_en.json; they are. 🙂